### ISSUE TITLE
Building fails as Cargo.lock uses sycamore-macro 0.8.0 (not the beta.7 version)

### ISSUE THREAD

**Issue by afiqzx** [framesurge/perseus#185]
This issue is reporting a bug in the code of Perseus. Details of the scope will be available in issue labels.
The author described their issue as follows:
> Building fails as Cargo.lock uses sycamore-macro 0.8.0 (not the beta.7 version)

The steps to reproduce this issue are as follows:
> I believe this bug is also reproducible using the base example in the documentattion.

- The author is willing to attempt a fix: true 

<details>
<summary>Tribble internal data</summary>

dHJpYmJsZS1yZXBvcnRlZCxDLWJ1ZyxBLWh5ZHJhdGlvbixhdXRob3Itd2lsbGluZy10by1pbXBs

</details>

### Additional information
Building the current beta.7 gives out the following error on my machine.
```
error: expected either `(` or `{` after element tag
   --> /home/afiq/.cargo/registry/src/github.com-1ecc6299db9ec823/sycamore-router-0.8.0-beta.7/src/router.rs:220:17
    |
220 |             view: props.view,
    |                 ^

```

After checking Cargo.lock and sycamore repo, I believe this happens because Cargo.lock file locked on sycamore-macro version "0.8.0", not the "0.8.0-beta.7". There is [this change](https://github.com/sycamore-rs/sycamore/pull/460/files#diff-da8d65e69f25ce45f4afbba09f3038530316b4c50f783acd604234570439129f) that breaks compilation.

I file this issue for anyone looking for a workaround, which is fixed by adding `sycamore-macro = "=0.8.0-beta.7"` into your Cargo.toml. I believe the true fix is by bumping the whole sycamore dependencies to stable 0.8.0.

**Comment by arctic-hen7** [comment]
Yes, I've already fixed this for the website, and I'm planning the full migration to Sycamore v0.8.0 today, which will also eliminate the last two wrapper `div`s. This should be fixed in a few hours!

**Comment by arctic-hen7** [comment]
All the necessary changes are now made on the `main` branch (except for the website), but I can't deploy a new version until Sycamore v0.8.1 is released, because there was a bug in the handling of `Children<'_, G>` that's only been fixed on Sycamore's `master` branch, and I can't deploy to crates.io with a Git patch dependency.

**Comment by afiqzx** [comment]
I saw that discussion in discord. Now we have to just wait for the blocking to resolve..

**Comment by arctic-hen7** [comment]
Latest Sycamore version is out, I'll finalize the changes in Perseus and release a new beta version in the next few hours.

**Comment by arctic-hen7** [comment]
As per the conversation on Discord, the new Sycamore version doesn't seem to actually fix this yet, possibly due to dependency versioning issues. Unfortunately, this will still be blocked until those (presumably upstream) issues are solved. (Alternatively, I may have stuffed up somewhere...)

**Comment by arctic-hen7** [comment]
Should be all fixed in v0.4.0-beta.8!
